packages/appstore/src/components/CFDs/cfd-real-accounts.tsx

```diff
@@ -0,0 +1,227 @@
+import AccountManager from '../account-manager';
+import AddDerived from 'Components/add-derived';
+import { TCFDAccountsProps, TPlatform, TDetailsOfEachMT5Loginid, TStaticAccountProps, TRootStore } from 'Types';
+import AddOptionsAccount from 'Components/add-options-account';
+import { useStores } from 'Stores/index';
+import { useHistory } from 'react-router-dom';
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+const CFDRealAccounts = ({
+    isDerivedVisible,
+    isFinancialVisible,
+    has_cfd_account_error,
+    current_list,
+    has_real_account,
+}: TCFDAccountsProps) => {
+    const { client, modules, common }: TRootStore = useStores();
+    const {
+        dxtrade_tokens,
+        setAccountType,
+        createCFDAccount,
+        enableCFDPasswordModal,
+        toggleJurisdictionModal,
+        disableCFDPasswordModal,
+        toggleMT5TradeModal,
+        setMT5TradeAccount,
+    } = modules.cfd;
+    const { setAppstorePlatform, platform } = common;
+    const { isEligibleForMoreRealMt5 } = client;
+    const history = useHistory();
+
+    const available_real_accounts: Array<TStaticAccountProps> = [
+        {
+            name: 'Derived',
+            description: localize('Trade CFDs on MT5 with Derived indices that simulate real-world market movements.'),
+            is_visible: isDerivedVisible(CFD_PLATFORMS.MT5),
+            disabled: has_cfd_account_error(CFD_PLATFORMS.MT5),
+            platform: CFD_PLATFORMS.MT5,
+            type: 'synthetic',
+        },
+        {
+            name: 'Financial',
+            description: localize('Trade CFDs on MT5 with forex, stocks & indices, commodities, and cryptocurrencies.'),
+            is_visible: isFinancialVisible(CFD_PLATFORMS.MT5),
+            disabled: has_cfd_account_error(CFD_PLATFORMS.MT5),
+            platform: CFD_PLATFORMS.MT5,
+            type: 'financial',
+        },
+        {
+            name: 'Deriv X',
+            description: localize(
+                'Trade CFDs on Deriv X with Derived indices, forex, stocks & indices, commodities and cryptocurrencies.'
+            ),
+            is_visible: isDerivedVisible(CFD_PLATFORMS.DXTRADE),
+            disabled: has_cfd_account_error(CFD_PLATFORMS.DXTRADE),
+            platform: CFD_PLATFORMS.DXTRADE,
+            type: 'synthetic',
+            // ToDo: deriv x should have type of all in new API
+            // type: 'all'
+        },
+    ];
+
+    const REAL_DXTRADE_URL = 'https://dx.deriv.com';
+    const DEMO_DXTRADE_URL = 'https://dx-demo.deriv.com';
+
+    const getDXTradeWebTerminalLink = (category: string, token?: string) => {
+        let url = category === 'real' ? REAL_DXTRADE_URL : DEMO_DXTRADE_URL;
+
+        if (token) {
+            url += `?token=${token}`;
+        }
+
+        return url;
+    };
+
+    const openAccountTransfer = (
+        data: DetailsOfEachMT5Loginid & { account_id?: string; platform?: string },
```

Comment:
```suggestion
        data: DetailsOfEachMT5Loginid & { account_id: string; platform: string },
```

Refer to the comments below